aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -175,26 +179,87 @@ static inline at::MemoryFormat mkldnn_convolution_memory_format(int64_t dims, bo
    return memory_format;
 }
 
-void _mkldnn_convolution_out (
+#define ONEDNN_PARAM_TYPE_CONV 0
+#define ONEDNN_PARAM_TYPE_LINEAR 1
+bool mkldnn_delete_param(int64_t param_handle, int64_t param_type) {
+  if(param_handle != 0) {
+    switch (param_type) {
+      case ONEDNN_PARAM_TYPE_CONV:
+        {
+          auto conv_params_ptr = (ideep::convolution_forward_params *) param_handle;
+          delete conv_params_ptr;
+          conv_params_ptr = nullptr;
+        }
+        break;
+      case ONEDNN_PARAM_TYPE_LINEAR:
+        {
+          auto linear_params_ptr = (ideep::inner_product_forward_params *) param_handle;
+          delete linear_params_ptr;
+          linear_params_ptr = nullptr;
+        }
+        break;
+      default:
+        return false;
```

Comment:
Should this throw an error?

Reply:
Here we can only do real delete operation when we recognized the type, and do nothing (in default path) but return false to tell the caller that there is something wrong in his calling that he should deal with. This should be safe enough?